codefori/vscode-ibmi
# Auth update notices

### Changes

* Corrects logic for when the password or private key gets updated
* Makes it clear to the user if the password or private is going to be used when it is changed.
* Adds en translation for the login and settings for a connection

### How to test this PR

Examples:

1. Update the password or private key and expect to see a message
2. See no errors in the login or login settings webviews regarding translation

### Checklist
